### PR TITLE
Feature: Multi-Language Switcher Improvements

### PR DESCRIPTION
**Pull request by ghost**
1. Do not show the current choosen language unless your want to choose for a new one.
2. Show tooltip when moving your mouse onto the language symbol for the choosen language.

### CODE REVIEW

**Comment by XhmikosR** [comment]
FYI this will conflict with #2392

**Comment by ghost** [comment]
@XhmikosR：Thanks, either you or I will meet with the conflict and fix it later, whatever submit is merged.

**Comment by XhmikosR** [comment]
BTW I was thinking the language switcher should be static in the templates.
Right now even if a page doesn't have a translation, the switcher shows all
of them.

Something to consider for the future maybe

**Comment by ghost** [comment]
> Right now even if a page doesn't have a translation, the switcher shows all of them.

Yes, but it would be nicer if we can show the current language as a tooltip for all of the users to see which language he/she is viewing with, and what's more, the picked language doesn't need to be shown in the menu list, unless you choose another one. 

That's my opition. For futher changes, maybe we can get together to change it little by little.

File: layouts/partials/header.hbs [hunk 1]
```diff
@@ -29,14 +29,13 @@
...
-            data-lang="{{locale}}"
-            title="{{languageEnglishVersion}}"
-          >
-            {{language}}
-          </li>
+        <li data-lang="{{locale}}" 
+        title="{{languageEnglishVersion}}" 
+        language="{{language}}">
+          {{language}}
+        </li>
```
Comment (alexandrtovmach) [inline layouts/partials/header.hbs]: Could you prettify the code

Comment (ghost) [reply layouts/partials/header.hbs]: ok, what about my changes? I've changed that.

Comment (ghost) [reply layouts/partials/header.hbs]: 捕获

**Review by alexandrtovmach — APPROVED** [review]

**Review by 3imed-jaberi — APPROVED** [review]

**Review by Aissaoui-Ahmed — APPROVED** [review]

**Pull request merged**